src/context/directory/handlers/organizations.js

```diff
@@ -35,6 +35,21 @@ async function dump(context) {
   organizations.forEach((organization) => {
     const organizationFile = path.join(organizationsFolder, sanitize(`${organization.name}.json`));
     log.info(`Writing ${organizationFile}`);
+
+    if (organization.connections.length > 0) {
+      organization.connections = organization.connections.map((c) => {
+        // connection is a computed field
+        const name = c.connection && c.connection.name;
+        delete c.connection_id;
+        delete c.connection;
+
+        return {
+          name,
+          ...c
+        };
```

Comment:
What do you think about this as a means to avoid mutating the referenced object in the mapper?

```suggestion
        const name = c.connection && c.connection.name;

        return {
          name,
          ...c
          connection_id: undefined,
          connection: undefined,
        };
```